### ISSUE TITLE
Why use class AdamW instead of "torch.optim.adamw "

### ISSUE THREAD

**Issue by Dawn90** [ngxbac/GAIN#1]
Hi,

Thanks a lot for sharing the pytorch version of GAIN!!
I'm trying to run the code and make some changes based on GAIN. I was wondering why you define the class AdamW by yourself instead of just import "torch.optim.adamw "? 

Many thanks!
Dong

**Comment by ngxbac** [comment]
Hi, 
At the time I did this project, I did not know the existence of `torch.optim.adamw` 
Thank you for pointing it out.

**Comment by Dawn90** [comment]
Thank you so much for your reply!

I have another question towards the line 88 in file 'gain.py'. I was wondering why you set the argument gradient=gradient here. It seems the original 'grad-cam' don't have this, while I haven't found corresponding formula in GAIN yet.
`grad_logits.backward(gradient=gradient, retain_graph=True)`

Many thanks!
Dong
